src/core/include/mp-units/framework/representation_concepts.h

```diff
@@ -313,12 +323,42 @@ concept Representation = detail::ScalarRepresentation<T> || detail::ComplexRepre
 
 namespace detail {
 
+#ifdef MP_UNITS_APPLE_CLANG_HACKS
```

Comment:
Do we need both `MP_UNITS_APPLE_CLANG_HACKS` and `MP_UNITS_XCODE15_HACKS`?

Reply:
Nope, this was a git mistake. Fixed.